Source/WebCore/page/LocalFrameView.cpp

```diff
@@ -3469,6 +3469,11 @@ void LocalFrameView::updateBackgroundRecursively(const std::optional<Color>& bac
                 view->layoutContext().scheduleLayout();
         }
     }
+
+    if (RenderView* renderView = this->renderView()) {
+        if (renderView->usesCompositing())
+            renderView->compositor().frameViewDidChangeBackground();
```

Comment:
I wonder if this should be moved to `::setBaseBackgroundColor` or handled via `setNeedsCompositingConfigurationUpdate`.

Would like to hear @smfr's thoughts on this.

Reply:
I agree that `setNeedsCompositingConfigurationUpdate` would be good to use, if it works.

Reply:
@pxlcoder how would I use setNeedsCompositingConfigurationUpdate to accomplish this?

Reply:
@pxlcoder fixed!

Reply:
oh, that didn't actually end up working...

Reply:
Hm, I can't reproduce locally, even after 200 iterations